Automattic/jetpack
# Add super-cache e2e tests to pnpm workspace

Super-cache e2e tests weren't in the pnpm workspace, so pnpm i wasn't working properly to install its deps. This PR adds it.

#### Changes proposed in this Pull Request:
* Add super-cache e2e to the pnpm workspace.

#### Other information:

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run `pnpm i`
* Ensure that it works, and that the super-cache e2e tests get their deps installed.
